### PR TITLE
Update manual instructions to deploy the Example Foundation

### PR DESCRIPTION
**Pull request by daniel-cit**
Update manual instructions to deploy the Example Foundation

- [X] 1-org
- [x] 2-environments
- [x] 3-networks-dual-svpc
- [x] 3-networks-hub-and-spoke
- [x] 4-projects
- [x] 5-app-infra

### CODE REVIEW

File: 1-org/README.md [hunk 1]
```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
...
-1. Run `./tf-wrapper.sh init production`.
+1. Use `terraform output` to get the Cloud Build project ID and the organization step Terraform Service Account from 0-bootstrap output. An environment variable `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` will be set using the Terraform Service Account to enable impersonation.
+   ```
+   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output cloudbuild_project_id | tr -d '"')
+   echo ${CLOUD_BUILD_PROJECT_ID}
+
+   export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT=$(terraform -chdir="../0-bootstrap/" output organization_step_terraform_service_account_email | tr -d '"')
+   echo ${GOOGLE_IMPERSONATE_SERVICE_ACCOUNT}
+   ```
+1. Run `./tf-wrapper.sh init production` .
```
Comment (bharathkkb) [inline 1-org/README.md:312]: nit: lets also make these easier to copy with fenced codeblocks.

File: 1-org/README.md [hunk 2]
```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
...
 We will now deploy our environment (production) using this script.
 When using Cloud Build or Jenkins as your CI/CD tool each environment corresponding to a branch is the repository for 1-org step and only the corresponding environment is applied.
 
 To use the `validate` option of the `tf-wrapper.sh` script, please follow the [instructions](https://cloud.google.com/docs/terraform/policy-validation/validate-policies#install) to install the terraform-tools component.
 
-1. Export the projects (`terraform-org-sa`) service account for impersonation `export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT="<IMPERSONATE_SERVICE_ACCOUNT>"`
-1. Run `./tf-wrapper.sh init production`.
+1. Use `terraform output` to get the Cloud Build project ID and the organization step Terraform Service Account from 0-bootstrap output. An environment variable `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` will be set using the Terraform Service Account to enable impersonation.
+   ```
+   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output cloudbuild_project_id | tr -d '"')
```
Comment (bharathkkb) [inline 1-org/README.md:306]: We can remove need for translate and use -raw instead
```suggestion
   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" -raw output cloudbuild_project_id)
```

File: 1-org/README.md [hunk 3]
```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
...
+   ```
+1. Change into `envs/shared` folder and rename `terraform.example.tfvars` to `terraform.tfvars`.
+   ```
+   cd envs/shared
+   mv terraform.example.tfvars terraform.tfvars
+   ```
+1. Update the file with values from your environment and 0-bootstrap output.
+1. Use `terraform output` to get the backend bucket and networks step Terraform Service Account (when using Hub and Spoke architecture) values from 0-bootstrap output.
+   ```
+   export backend_bucket=$(terraform -chdir="../../../0-bootstrap/" output gcs_bucket_tfstate | tr -d '"')
```
Comment (felipecrescencio-cit) [inline 1-org/README.md:284]:
```suggestion
   export backend_bucket=$(terraform -chdir="../../../0-bootstrap/" output -raw gcs_bucket_tfstate)
```
`raw` output option must be after `output`

File: 1-org/README.md [hunk 4]
```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
...
+    echo "backend_bucket = ${backend_bucket}"
+
+   networks_step_terraform_sa_email=$(terraform -chdir="../../../0-bootstrap/" output networks_step_terraform_service_account_email | tr -d '"')
```
Comment (felipecrescencio-cit) [inline 1-org/README.md:287]:
```suggestion
   networks_step_terraform_sa_email=$(terraform -chdir="../../../0-bootstrap/" output -raw networks_step_terraform_service_account_email)
```

File: 1-org/README.md (hunk 2 above)
Comment (felipecrescencio-cit) [reply 1-org/README.md:306]:
```suggestion
   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output -raw cloudbuild_project_id)
```
`-raw` option must be after `output`command

File: 1-org/README.md [hunk 5]
```diff
@@ -267,29 +267,56 @@ to run the command as the Terraform service account.
...
 To use the `validate` option of the `tf-wrapper.sh` script, please follow the [instructions](https://cloud.google.com/docs/terraform/policy-validation/validate-policies#install) to install the terraform-tools component.
 
-1. Export the projects (`terraform-org-sa`) service account for impersonation `export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT="<IMPERSONATE_SERVICE_ACCOUNT>"`
-1. Run `./tf-wrapper.sh init production`.
+1. Use `terraform output` to get the Cloud Build project ID and the organization step Terraform Service Account from 0-bootstrap output. An environment variable `GOOGLE_IMPERSONATE_SERVICE_ACCOUNT` will be set using the Terraform Service Account to enable impersonation.
+   ```
+   export CLOUD_BUILD_PROJECT_ID=$(terraform -chdir="../0-bootstrap/" output cloudbuild_project_id | tr -d '"')
+   echo ${CLOUD_BUILD_PROJECT_ID}
+
+   export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT=$(terraform -chdir="../0-bootstrap/" output organization_step_terraform_service_account_email | tr -d '"')
```
Comment (felipecrescencio-cit) [inline 1-org/README.md:309]:
```suggestion
   export GOOGLE_IMPERSONATE_SERVICE_ACCOUNT=$(terraform -chdir="../0-bootstrap/" output -raw  organization_step_terraform_service_account_email)
```

File: 2-environments/README.md [hunk 6]
```diff
@@ -208,33 +208,56 @@ commands. The `-T` flag is needed for Linux, but causes problems for MacOS.
...
+   ```
+1. Rename `terraform.example.tfvars` to `terraform.tfvars`.
+   ```
+   mv terraform.example.tfvars terraform.tfvars
+   ```
+1. Update the file with values from your environment and 0-bootstrap output.See any of the envs folder [README.md](./envs/production/README.md#inputs) files for additional information on the values in the `terraform.tfvars` file.
+1. Use `terraform output` to get the backend bucket value from 0-bootstrap output.
+   ```
+   export backend_bucket=$(terraform -chdir="../0-bootstrap/" output gcs_bucket_tfstate | tr -d '"')
+    echo "backend_bucket = ${backend_bucket}"
```
Comment (felipecrescencio-cit) [inline 2-environments/README.md:225]:
```suggestion
   export backend_bucket=$(terraform -chdir="../0-bootstrap/" -raw output gcs_bucket_tfstate)
   echo "backend_bucket = ${backend_bucket}"
```

File: 1-org/README.md (hunk 2 above)
Comment (daniel-cit) [reply 1-org/README.md]: fixed

File: 1-org/README.md (hunk 1 above)
Comment (daniel-cit) [reply 1-org/README.md]: inline codeblocks replaced with fenced codeblocks

File: 1-org/README.md [hunk 7]
```diff
@@ -267,14 +267,22 @@ to run the command as the Terraform service account.
...
+   cd 1-org
+   cp ../build/tf-wrapper.sh .
+   chmod 755 ./tf-wrapper.sh
+   ```
+1. Change into `envs/shared` folder and rename `terraform.example.tfvars` to `terraform.tfvars`.
+   ```
+   cd envs/shared
+   mv terraform.example.tfvar terraform.tfvar
+   ```
+1. Update the file with values from your environment and bootstrap. Use `terraform output` to get values 0-bootstrap output.
```
[the remark this answers is not in the document]
Comment (daniel-cit) [reply 1-org/README.md]: done

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**